recipes/nfir/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+from conan import ConanFile
+from conan.tools.cmake import cmake_layout, CMake, CMakeToolchain
+from conan.tools.files import copy, get, collect_libs
+from conan.tools.files import apply_conandata_patches
+from os import path
+
+class NFIRConan(ConanFile):
+    name = "nfir"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/usnistgov/NFIR"
+    topics = ("NIST, NFIR")
+    license = "NIST"
+    description = "The NIST Fingerprint Image Resampler NFIR is a library capable of upsampling or downsampling fingerprint images."
+    settings = "os", "compiler", "build_type", "arch"
+    options = { "shared": [True, False] }
+    default_options = { "shared": False }
```

Comment:
```suggestion
    options = { "shared": [True, False], "fPIC": [True, False] }
    default_options = { "shared": False, "fPIC": True }
```
`fPIC` option needs to be provided for `library` package type.